app/src/main/res/values/strings.xml

```diff
@@ -48,7 +48,7 @@
     <!--<string name="url_motivate_video" translatable="false">http%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DgJscrxxl_Bg</string>-->
```

Comment:
Better to remove commented out code

Reply:
Will be fixed https://github.com/mgolokhov/dodroid/issues/139